common/identity.go

```diff
@@ -47,6 +48,12 @@ func IsUserPresent(ctx context.Context) bool {
 
 }
 
+// IsFlorenceIdentityPresent determines if a florence identity is present on the given context
+func IsFlorenceIdentityPresent(ctx context.Context) bool {
```

Comment:
Just wonder if it might be useful to also return the token if it exists?

Reply:
I should probably have created a separate function to retrieve the token from context